addon/mixins/ajax-request.js

```diff
@@ -615,9 +616,9 @@ export default Mixin.create({
    */
   generateDetailedMessage(status, headers, payload, requestData) {
     let shortenedPayload;
-    const payloadContentType = headers['Content-Type'] || 'Empty Content-Type';
+    const payloadContentType = getHeader(headers, 'Content-Type') || 'Empty Content-Type';
 
-    if (payloadContentType === 'text/html' && payload.length > 250) {
+    if (payloadContentType.toLowerCase() === 'text/html' && payload.length > 250) {
```

Comment:
Case-insensitive content-type check. This is technically "extra" but related-enough that I couldn't help but address it as well.